README.md

```diff
@@ -1,4 +1,7 @@
-# DeGirum PySDK Examples
+# DeGirum PySDK 
+
+DeGirum PySDK is an edge AI deployment focused software development kit that supports multiple hardware options, enabling AI solution providers to add AI to their applications using simple APIs.
```

Comment:
May better to rephrase like this:

DeGirum PySDK is software development kit, focused on edge AI deployment, that supports ...

Otherwise too many chained nouns, difficult to grasp an object.

Reply:
Agree. Fixed it.